cef/hudkit_window.cc

```diff
@@ -52,6 +52,7 @@ void HudkitWindow::Initialize()
         gtk_window_set_accept_focus(gtkWindow, false);
         gtk_window_set_decorated(gtkWindow, false);
         gtk_window_set_resizable(gtkWindow, false);
+        lockedState = true;
```

Comment:
Was this an intentional addition?

Reply:
Yeah, this block (not used currently) locks the window on creation, so lockedState should be true if we start it locked